docs/en_US/Compressor/CompressionUtils.md

```diff
@@ -0,0 +1,100 @@
+
+### Usage
+
+Following codes show the basic usage of the SensitivityAnalysis.
+```python
+from nni.compression.analysis_utils.sensitivity.torch.sensitivity_analysis import SensitivityAnalysis
+
+def val(model):
+    model.eval()
+    total = 0
+    correct = 0
+    with torch.no_grad():
+        for batchid, (data, label) in enumerate(val_loader):
+            data, label = data.cuda(), label.cuda()
+            out = model(data)
+            _, predicted = out.max(1)
+            total += data.size(0)
+            correct += predicted.eq(label).sum().item()
+    return correct / total
+
+s_analyzer = SensitivityAnalysis(model=net, val_func=val)
+sensitivity = s_analyzer.analysis(val_args=[net])
+os.makedir(outdir)
+s_analyzer.export(os.path.join(outdir, filename))
+s_analyzer.visualization(outdir, merge=True)
+s_analyzer.visualization(outdir, merge=False)
+```
+
+Two key parameters of SensitivityAnalysis are model, and val_func. 'model' is the neural network that to be analyzed and the 'val_func' is the validation function that returns the model accuracy on the validation dataset. Due to different scenarios may have different ways to calculate the loss/accuracy, so users should prepare a function that returns the model accuracy on the dataset and pass it to SensitivityAnalysis.
+SensitivityAnalysis can export the sensitivity results as a csv file and visualization the sensitivity results of each layer in the model, usage is shown in the example above.
+
+Futhermore, users can specify the sparsities values used to prune for each layer by optinal parameter 'sparsities'.
+```python
+s_analyzer = SensitivityAnalysis(model=net, val_func=val, sparsities=[0.25, 0.5, 0.75])
+``` 
+the SensitivityAnalysis will prune 25% 50% 75% weights gradually for each layer, and record the model's accuracy at the same time (SensitivityAnalysis only prune a layer once a time, the other layers are set to their original weights). If the sparsities is not set, SensitivityAnalysis will use the numpy.arange(0.1, 1.0, 0.1) as the default sparsity values.
+
+Users can also speed up the progress of sensitivity analysis by the early_stop option. By default, the SensitivityAnalysis will test the accuracy under all sparsities for each layer. In contrast, when the early_stop is set, the sensitivity analysis for a layer will stop, when the accuracy drop already reaches the threshold set by early_stop.
+```python
+s_analyzer = SensitivityAnalysis(model=net, val_func=val, sparsities=[0.25, 0.5, 0.75], early_stop=0.1)
```

Comment:
If early_stop is supported,  does the tool support configuring `maximize` / `minimize` mode of metrics ?

Reply:
Yes, of course, supporting the maximize/minimize model also only needs a negligible effort. My only concern is whether too many choices(early_stop&minimize/maximize) will confuse users.

Reply:
It's reasonable to allow user to set optimization mode,  otherwise only supporting maximize is too limited.  Suggest to replace `accuracy` with general term such as `metrics` or other term, and support maximize/minimize.

Reply:
Yeah, I'll update the code. Thanks for the advice.